ganga/GangaCore/Core/GangaRepository/GangaRepositoryJSON.py

```diff
@@ -0,0 +1,4 @@
+"""
+Currently a work in progress
+
+"""
```

Comment:
Remember a newline at the end of all files. You might be able to set your editor to add one automatically.